File: Python/tower_of_Hanoi.py
def towerOfHanoi(n, from_, to, aux):
    if n==1:
        print(from_,"------>", to)
        return
    towerOfHanoi(n-1, from_, aux, to)
    print(from_,"------>", to)
    towerOfHanoi(n-1, aux, to, from_)

n = int(input("Enter the number of disks:  "))
towerOfHanoi(n, "A", "B", "C")

File: C/tower_of_hanoi.c
#include<stdio.h>

void towerOfHanoi(int n, char from, char aux, char to){
    if(n==1){
        printf("%c ------> %c\n", from, to);
        return;
    }
    towerOfHanoi(n-1, from, to, aux);
    printf("%c ------> %c\n", from, to);
    towerOfHanoi(n-1, aux, from, to);
    return;
}

int main(void){
    int n;
    printf("Enter the number of Disks: ");
    scanf("%d",&n);
    towerOfHanoi(n, 'A', 'C', 'B');
    return 0;
}